config/locales/en.yml

```diff
@@ -418,20 +444,48 @@ en:
             href: "https://www.gov.uk/government/news/government-agrees-measures-with-energy-industry-to-support-vulnerable-people-through-covid-19"
           - text: "What to do if you cannot pay your rent or mortgage (Shelter)"
             href: "https://england.shelter.org.uk/housing_advice/coronavirus"
+            show_to_nations:
+              - England
+          - text: "What to do if you cannot pay your rent or mortgage (Shelter Scotland)"
+            href: "https://scotland.shelter.org.uk/get_advice/scottish_housing_advice_coronavirus_COVID_19"
+            show_to_nations:
+              - Scotland
+          - text: "What to do if you cannot pay your rent or mortgage (Shelter Cymru)"
+            href: "https://sheltercymru.org.uk/get-advice/coronavirus/"
+            show_to_nations:
+              - Wales
           - text: "What to do if you cannot pay your rent, mortgage or bills (Citizens Advice)"
             href: "https://www.citizensadvice.org.uk/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/"
+            show_to_nations:
+              - England
+          - text: "What to do if you cannot pay your rent, mortgage or bills (Citizens Advice Scotland)"
+            href: "https://www.citizensadvice.org.uk/scotland/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/"
+            show_to_nations:
+              - Scotland
+          - text: "What to do if you cannot pay your rent, mortgage or bills (Citizens Advice)"
```

Comment:
Change to "What to do if you cannot pay your bills or mortgage (Citizens Advice)"

Reason - little information here about rent

Reply:
👍